camelot/view/controls/editors/one2manyeditor.py

```diff
@@ -148,6 +148,8 @@ def button_clicked(self, checked):
     def set_right_toolbar_actions(self, action_routes, toolbar):
         if action_routes is not None:
             for route_with_render_hint in action_routes:
+                if not isinstance(route_with_render_hint, RouteWithRenderHint):
```

Comment:
in principe is RouteWithRenderHint een server side class, dus zou die in principe in de ui nt moeten voorkomen
(wat uiteraard nt hoeft overeen te komen met de realiteit :) )